pppd/lcp.c

```diff
@@ -152,6 +163,8 @@ static struct option lcp_option_list[] = {
       "Set time in seconds between LCP echo requests", OPT_PRIO },
     { "lcp-echo-adaptive", o_bool, &lcp_echo_adaptive,
       "Suppress LCP echo requests if traffic was received", 1 },
+    { "lcp-rtt-file", o_string, &lcp_rtt_file,
+      "Write to this file the RTT of LCP echo requests", OPT_PRIO },
```

Comment:
Suggest "Filename for logging the round-trip time of LCP echo requests" or similar.